test/HardwareObjectsMockup.xml/mxcube-web/server.yaml

```diff
@@ -14,3 +14,8 @@ mxcube:
 
   usermanager:
     class: UserManager
+    inhouse_is_staff: True
+    users:
+      -
+        username: opid0
+        role: staff
```

Comment:
It's only possible to give a user one role at the moment, one could consider making this a list